PowerEditor/installer/nativeLang/swedish.xml

```diff
@@ -305,7 +306,8 @@
 					<Item id="50006" name="Sökvägskomplettering"/>
 				</Commands>
 			</Main>
-			<Splitter />
+            <Splitter>
+            </Splitter>
```

Comment:
@magol OK, please do a major change in this PR, then afterward only the necessary modification afterward.